docuilib/src/components/MainSection.tsx

```diff
@@ -22,7 +24,6 @@ export default () => {
     const data = await response.json();
     const _starsCount = (data.stargazers_count / 1000).toFixed(1);
 
-
     localStorage.setItem(STARS_COUNT_KEY, _starsCount.toString());
     setStarsCount(_starsCount);
```

Comment:
Stars still showing `NanK`, the value should come form the config file if _starsCount doesn't have any value.

Reply:
I added a fallback to handle this use case